src/main/java/org/kainos/ea/db/HelloWorldDao.java

```diff
@@ -0,0 +1,34 @@
+package org.kainos.ea.db;
+
+import org.kainos.ea.cli.HelloWorld;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class HelloWorldDao {
+
+    public List<HelloWorld> getHelloWorld() throws SQLException {
+        Connection c = DatabaseConnector.getConnection();
+        assert c != null;
+
+        Statement st = c.createStatement();
```

Comment:
This can be put into a `try-with-resource` as there is a potential for the connection to never get closed